src/accelerate/state.py

```diff
@@ -27,6 +35,18 @@
     _tpu_available = False
 
 
+def env2int(env_list, default=-1):
```

Comment:
The name is a bit too short, and we don't use 2 like this in this repo (for instance we have `send_to_device` not `send2device`). It also returns the first match What about `get_int_from_env`?

`env_list` is also not the right name for the argument as we don't pass a list of envs, but a lsit of names to query in the env, so I would use `env_keys` instead of `env_list`. `default` should not have a default value since there is not a canonical one (the function is used with -1 once, 0 twice and 1 twice below).

Lastly, a short docstring that explains what this function does (returning the first positive env found in the list or the default) would be awesome!